docs/lang/articles/visualization/ggui.md

```diff
@@ -118,6 +325,55 @@ You can render a scene on a canvas.
 canvas.scene(scene)
 ```
 
+### Fetching Color/Depth information
+
+```python
+img = window.get_image_buffer()
+window.get_depth_buffer(scene_depth)
+depth = window.get_depth_buffer_as_numpy()
+```
+
+After rendering the current scene, you can fetch the color and depth information of the current scene using `get_image_buffer()` and `get_depth_buffer_as_numpy()`, which copy the gpu data to a NumPy array(cpu).
+`get_depth_buffer()` copies the GPU data to a Taichi field (depend on the `arch` you choose) or copies data from GPU to GPU.
```

Comment:
Try:
> After a call to `canvas.scene()`, the rendered color and depth data can be dumped to a NumPy array with `get_image_buffer()` and `get_depth_buffer_as_numpy()`. Depth data can also be directly copied to an Taichi field with `get_depth_buffer()`.

btw why `get_image_buffer()` targets numpy but `get_depth_buffer()` targets Taichi field? better to fix this inconsistency.

Reply:
I talked about the function's name with Ailing, and we decided to name it as this to clearly tell the user what data type it would target.

Reply:
well the key is that we should keep the names in accordance so that the user can be intuitively guess or make assumption about the outcome of another. cc @ailzhang

Reply:
@Morcki can we rename `get_image_buffer` to `get_image_buffer_as_numpy` in this PR to keep them consistent?

Reply:
already done.